app/models/grda_warehouse/hud/enrollment.rb

```diff
@@ -163,6 +163,8 @@ class Enrollment < Base
       visible_to(user)
     end
 
+    # this scope is defined in the ClientAccessControl::GrdaWarehouse::Hud::EnrollmentExtension
+    # TODO: why not just define it here
```

Comment:
Initially we envisioned multiple possible `visible_to` mechanisms that might be built out in different drivers and enabled as necessary.  So far that hasn't manifested, so the main reason for defining it in the driver is that drivers should be dependent on the app, not the app on the driver.